privacyidea/lib/tokens/emailtoken.py

```diff
@@ -542,6 +542,7 @@ def enroll_via_validate(cls, g, content, user_obj):
                 "type": token_obj.type,
                 "message": _("Please enter your new email address!")}
         detail["multi_challenge"] = [chal]
+        detail.update({"preferred_client_mode": "interactive"})
```

Comment:
Please always use the constant: CLIENTMODE.INTERACTIVE,

Reply:
https://github.com/privacyidea/privacyidea/pull/3457/commits/2fd9852e28eb917cf3fe220175f73fed2ea99ef9